addons/mail/static/src/core/common/chat_hub_model.js

```diff
@@ -146,9 +146,15 @@ export class ChatHub extends Record {
         );
     }
 
+    showBubbles = fields.Attr(true, {
+        compute() {
+            return !this.store.discuss?.isActive;
+        },
+    });
+
     showConversations = fields.Attr(false, {
         compute() {
-            return this.canShowOpened.length + this.canShowFolded.length > 0;
+            return this.showBubbles && this.canShowOpened.length + this.canShowFolded.length > 0;
```

Comment:
Keep diff like before: it shouldn't be based on `showBubbles`. It's the opposite that should be done: `showBubbles` is based on other field values.